src/entities/ring-entity.ts

```diff
@@ -196,10 +197,17 @@ export class Ring {
   })
   public accuracyOfPullusAge: AccuracyOfPullusAge;
 
-  // Related fields in access 'Lat deg', 'Lat min', 'Lat sec', 'Lon deg', 'Lon min', 'Lon sec',
-  @Length(15, 15, { message: equalLength(15) })
-  @Column('varchar', { nullable: true, default: null })
-  public geographicalCoordinates: string | null;
+  // Related fields in access 'Lat deg', 'Lat min', 'Lat sec'
+  @IsOptional()
+  @IsNumber()
+  @Column('numeric', { nullable: true, default: null })
+  public latitude?: number | null;
+
+  // Related fields in access 'Lon deg', 'Lon min', 'Lon sec'
+  @IsOptional()
+  @IsNumber()
```

Comment:
please validate these values by `max` and `min` value

Reply:
Done